internal/storage/image.go

```diff
@@ -315,7 +315,7 @@ func (svc *imageService) ListImages(systemContext *types.SystemContext, filter s
 				return nil, err
 			}
 			results, err = svc.appendCachedResult(systemContext, ref, image, results, newImageCache)
-			if err != nil {
+			if err != nil && !strings.Contains(err.Error(), fmt.Sprintf(`error locating item named "%s"`, "manifest")) {
```

Comment:
This is too specific. Another option we discussed is keeping a list of images being pulled and then if we get any such error for an image in that list, we skip it.